lib/matplotlib/axes/_axes.py

```diff
@@ -326,13 +326,27 @@ def inset_axes(self, bounds, *, transform=None, zorder=5, **kwargs):
             Defaults to `ax.transAxes`, i.e. the units of *rect* are in
             Axes-relative coordinates.
 
+        projection : {None, 'aitoff', 'hammer', 'lambert', 'mollweide', \
+'polar', 'rectilinear', str}, optional
+            The projection type of the inset `~.axes.Axes`. *str* is the name
+            of a custom projection, see `~matplotlib.projections`. The default
+            None results in a 'rectilinear' projection.
+
+        polar : bool, default: False
+            If True, equivalent to projection='polar'.
```

Comment:
We can't remove `polar` because it comes through the internal meothd, but I wonder if we want to not document it, since it duplicates `projection`? @timhoffm

I recall discussing it, and it couldn't really be dropped from normal since it was so widely used, but since this is new, we don't have to worry about existing users.

Reply:
Unfortunately, `polar` is a redundant API. OTOH `polar` is widely used and we have not even discouraged its use, I consider it proper first-class API. One can regard it as a shortcut.
Overall, it's there and it's valid to be used. Therefore I recommend to also document it.

Reply:
I’m not sure where to go from here as there are differing opinions.  FWIW I think it’s intuitive that the various axes creation functions should take the same keywords.  So if I were used to using `polar` with the other functions I would expect to use it with this one too.  I’m not even sure whether I would check the docstring before trying it!

Reply:
Picking up my API authority hat, I decide that the present consistent way is the desired one. If we want to do something about `polar`, that'd need to be a separate discussion and a global change (but I advise against that).